File: uri 1037.py
values = float(input())
if values >= 0 and values <= 25:
    print("Intervalo [0,25]")
elif values > 25 and values <= 50:
    print("Intervalo (25,50]")
elif values > 50 and values <= 75:
    print("Intervalo (50,75]")
elif values > 75 and values <= 100:
    print("Intervalo (75,100]")
else:
    print("Fora de intervalo")

File: uri 1037.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float a;
    scanf("%f",&a);
    if(a>=0 && a<=100){
        if(a<=25.0){
            printf("Intervalo [0,25]\n");
        }
        else if(a<=50.0){
            printf("Intervalo (25,50]\n");
        }
        else if(a<=75.0){
            printf("Intervalo (50,75]\n");
        }
        else{
            printf("Intervalo (75,100]\n");
        }
    }
    else{
        printf("Fora de intervalo\n");
    }

    return 0;
}
